src/config/bin_package.rs

```diff
@@ -90,7 +90,8 @@ impl BinPackage {
             };
             let mut file = config.bin_target_dir.as_ref()
                 .map(|dir| dir.into())
-                .unwrap_or_else(|| metadata.rel_target_dir().join("server"));
+                //Can't use absolute path because the path gets stored in snapshot testing, and it differs between developers
```

Comment:
Could you please update to follow the rust fmt guidelines? (space after the opening slashes).